ios/Capacitor/Capacitor/CAPBridgeViewController.swift

```diff
@@ -89,15 +89,21 @@ public class CAPBridgeViewController: UIViewController, CAPBridgeDelegate, WKScr
     }
     webView?.configuration.preferences.setValue(true, forKey: "allowFileAccessFromFileURLs")
     view = webView
-    
+
     setKeyboardRequiresUserInteraction(false)
-    
+
     bridge = CAPBridge(self, o, capConfig, specifiedScheme)
 
+    webView?.isOpaque = false
```

Comment:
what is this supposed to do? shouldn't it be reset to the default value after the splash disappears?